_pages/en/it-picture-medium-term.md

```diff
@@ -26,124 +26,131 @@ The foundational pieces for "Government as a service" vision are available and i
-- https://cloud.google.com/solutions/devops/devops-tech-test-data-management
-- https://cloud.google.com/solutions/devops/devops-tech-shifting-left-on-security
-
-Teams are able to provision an environment or device in an automated fashion, and know that any new environment or device reproduced from the same configuration is identical.
+- [https://cloud.google.com/solutions/devops/devops-tech-version-control](https://cloud.google.com/solutions/devops/devops-tech-version-control)
+- [https://cloud.google.com/solutions/devops/devops-tech-continuous-integration](https://cloud.google.com/solutions/devops/devops-tech-continuous-integration)
+- [https://cloud.google.com/solutions/devops/devops-tech-deployment-automation](https://cloud.google.com/solutions/devops/devops-tech-deployment-automation)
+- [https://cloud.google.com/solutions/devops/devops-tech-trunk-based-development](https://cloud.google.com/solutions/devops/devops-tech-trunk-based-development)
+- [https://cloud.google.com/solutions/devops/devops-tech-test-automation](https://cloud.google.com/solutions/devops/devops-tech-test-automation)
+- [https://cloud.google.com/solutions/devops/devops-tech-architecture](https://cloud.google.com/solutions/devops/devops-tech-architecture)
+- [https://cloud.google.com/solutions/devops/devops-tech-teams-empowered-to-choose-tools](https://cloud.google.com/solutions/devops/devops-tech-teams-empowered-to-choose-tools)
+- [https://cloud.google.com/solutions/devops/devops-tech-test-data-management](https://cloud.google.com/solutions/devops/devops-tech-test-data-management)
+- [https://cloud.google.com/solutions/devops/devops-tech-shifting-left-on-security](https://cloud.google.com/solutions/devops/devops-tech-shifting-left-on-security)
+
+Teams are able to provision an environment or device in an automated fashion, and know that any new environment or device generated from the same configuration is identical.
 Production services can be restored repeatedly and predictably even when catastrophic events occur.
+Chaos Engineering is being considered for use to build confidence in systems' capabilities to withstand unexpected conditions.
 
-Source code, assets, configurations, or other documents part of a project's development and deployment are stored using a version control system that records changes to files.
-Changes triggers automated tests and a build process that creates packages (executable, container, VM, ..) that can be deployed to any environment (testing or production) in an automated way.
+Source code, assets, configurations, or other documents part of a project's development and deployment are stored in a version control system that records changes to files.
+Changes trigger automated tests and a build process that creates packages (executable, container, virtual machine, ...) that can be deployed to any environment (testing or production) in an automated way.
 Operations and infrastructure have moved towards automating as much as possible.
```

Comment:
Would propose this addition on line 119:

"[...] as much as possible, able to shift workloads and runtime from one service provider to another as it needs to."

Reply:
Good one, done :)